docs/sources/reference/run.md

```diff
@@ -312,6 +312,7 @@ container:
     -m="": Memory limit (format: <number><optional unit>, where unit = b, k, m or g)
     -memory-swap="": Total memory limit (memory + swap, format: <number><optional unit>, where unit = b, k, m or g)
     -c=0 : CPU shares (relative weight)
+    --cpu-quota=0: CPU cfs quota
```

Comment:
Limit the CPU CFS [Complete Fair Scheduler](http://en.wikipedia.org/wiki/Completely_Fair_Scheduler) quota. The default value of 0 is equivalent to 100% or 100000.